### ISSUE TITLE
ansi colors

### ISSUE THREAD

**Issue by pjv** [Fred-Barclay/Termination#48]
seems like the new "themification" makes my custom ansi colors from the packages settings be ignored.

is there a way to make my own theme for the ansi colors?

or is there a way to make the package pay attention to my custom ansi colors from the settings?

**Comment by Fred-Barclay** [comment]
Yep, sorry, the goal was to make each theme handle colours rather than having a universal set of ansi colours for the themes.
You could use custon ansi colours before this last update, right? Can you walk me through exactly how you go about that? I'll see if I can't add this functionality back. :)

Thanks for trying Termination!

**Comment by pjv** [comment]
If you look at the package's settings, you'll see that there is the ability to specify "Ansi Colors" in there. Without looking at your commits, I imagine that somewhere along the line in getting ansi colors from themes, you either nixed looking at the colors specified in the settings, or are overriding them by getting the colors from the theme after the colors from the settings.

**Comment by Fred-Barclay** [comment]
That would be this commit ae4dee7269471a1448c2f3d23996448d4a978d66 😄 
I'll try and straighten it out.
